conf/application.conf

```diff
@@ -202,4 +202,6 @@ nino-logging.enabled = true
 # if enabled the account creation cap check is done before the DES Eligibility Check API call
 enable-early-cap-check = true
 
-gov-uk.url="https://www.gov.uk"
\ No newline at end of file
+gov-uk.url="https://www.gov.uk"
+
+gov-uk.not-eligible.url="https://www.tax.service.gov.uk/help-to-save/not-eligible"
```

Comment:
Not sure if this is needed, can we not just direct the user directly to our page?